packages/site/src/assets/JobberLogo.svg.tsx

```diff
@@ -1,12 +1,18 @@
+        fill="#012939"
```

Comment:
`fill` should be using a semantic token - hardcoding to #012939 breaks in dark mode:
![image](https://github.com/user-attachments/assets/800e4c0a-7534-4472-8f3a-0fce653b8591)

**Before** (using `--surface--reverse`)
![image](https://github.com/user-attachments/assets/490193e7-bc9b-45c9-8cf0-aa9ec7e5daa7)

**After**
![image](https://github.com/user-attachments/assets/cfe36aff-6781-4bda-97a3-e6e2478a352f)

Reply:
oops! I literally copied it from Figma 🙈  fixed